third_party/terraform/data_sources/data_source_google_iam_policy.go.erb

```diff
@@ -39,7 +41,10 @@ func dataSourceGoogleIamPolicy() *schema.Resource {
 						"members": {
 							Type:     schema.TypeSet,
 							Required: true,
-							Elem:     &schema.Schema{Type: schema.TypeString},
+							Elem:     &schema.Schema{
+								Type:         schema.TypeString,
+								ValidateFunc: validation.StringDoesNotMatch(regexp.MustCompile("^deleted:"), "Terraform does not support deleted IAM members"),
```

Comment:
Just to make it super extra clear, maybe something like "Terraform does not support IAM bindings for deleted users", so people don't get the wrong impression that they can't delete people from their policies.